python/lsst/pipe/tasks/hips.py

```diff
@@ -429,6 +447,24 @@ def _make_cli_parser(cls):
                 help="Data ID expression used when querying for input coadd datasets.",
             )
 
+        parser_build.add_argument(
+            "--output",
+            type=str,
+            help="Name of the output CHAINED collection.",
```

Comment:
Mention that if this is given without output_run that a default timestamped output run will be used?